db/migrate/20200115141035_add_subject_knowledge_enhancement_available_to_financial_incentive.rb

```diff
@@ -3,7 +3,7 @@ def change
     add_column :financial_incentive, :subject_knowledge_enhancement_course_available, :boolean, null: false, default: false
 
     say_with_time "populating finanical incentive subject knowledge enhancement course available" do
-      Subjects::FinancialIncentiveSetSubjectKnowledgeEnhancementCourseAvailableService.new.execute
+      Subjects::FinancialIncentiveSetSubjectKnowledgeEnhancementCourseAvailableService.new(year: 2020).execute
```

Comment:
why are we changing historical migration?

Reply:
Its zero operation, just code alignment